src/components/InputDropdown/index.tsx

```diff
@@ -12,9 +12,11 @@ import {
   DropdownErrorText,
   DropdownInput,
   DropdownInputContainer,
+  DropdownInputFrame,
```

Comment:
agh just pulled and tested and for some reason it seems like this issue is still present for me....(i.e. dropdown clickable outside of the box)

Reply:
should probably get everyone to test during tue worksesh, i will add to agenda